server/plugins/irc-events/message.ts

```diff
@@ -26,6 +27,11 @@ export default <IrcEventHandler>function (irc, network) {
 		handleMessage(data);
 	});
 
+	irc.on("tagmsg", function (data) {
+		data.type = MessageType.TAGMSG;
+		handleMessage(data);
```

Comment:
I don't think it makes sense to reuse a function if you don't actually want to do the same things... you exit early and need to add a bunch of special cases that would be better put into its own function.

You do not in fact want to treat tagmessages remotely the same way as privmsg, action et al, so in my opinion why even try?

Reply:
Right, good point.

Reply:
Fixed, I just put a comment for now until there's a user of TAGMSG handlers.